hnn_core/gui/_viz_manager.py

```diff
@@ -150,7 +150,7 @@ def _update_ax(fig, ax, single_simulation, sim_name, plot_type, plot_config):
 
     elif plot_type == 'PSD':
         if len(dpls_copied) > 0:
-            color = next(ax._get_lines.prop_cycler)['color']
+            color = ax._get_lines.get_next_color()
```

Comment:
Does this API change work for earlier versions of matplotlib as well? If so I'm definitely happy to merge as is!

Reply:
Yep, I developed with 3.8.2 and tested it with matplotlib older versions 3.5.3 and 3.7.4.  ">=3.5.3" is what is specified in the setup file.